_resources/_styles/landingpage.styl

```diff
@@ -0,0 +1,132 @@
+.FlagImgContainer {
+  background: url("/_img/people_only.png");
+  background-repeat: no-repeat;
+  background-size:100%;
+  height: 35vh;
+  width: 100%;
+}
+  @media laptop {
+    .FlagImgContainer {
+      background: url("/_img/people_flag.png") no-repeat;
+      background-position: center center;
+      background-size:100%;
```

Comment:
missing space. i might not catch all of these but it's good to be consistent here.